src/server/modules/user/UserController.ts

```diff
@@ -184,7 +214,9 @@ export class UserController {
       sortDirection = 'desc',
       isFile,
       state,
+      tags = '',
     } = req.query
+    const tagList = tags ? tags.toString().toLowerCase().split(';') : []
```

Comment:
should we add `trim()` to preempt any trailing whitespace?

Reply:
I thought about it too but that can't happen since we already have the Joi validation